auth-jwt-service/src/main/java/io/stargate/auth/jwt/AuthnJwtService.java

```diff
@@ -69,7 +68,7 @@ public String createToken(String key, Map<String, String> headers) {
    */
   @Override
   public AuthenticationSubject validateToken(String token) throws UnauthorizedException {
-    if (Strings.isNullOrEmpty(token)) {
+    if ((token == null) || token.isEmpty()) {
```

Comment:
Normally I'm all for using helper methods, but here replacement allows us to get rid of the whole Guava dep (shaded or not). Ditto for isolated cases of removing "@VisibleForTesting".